tock/hours/views.py

```diff
@@ -479,7 +479,8 @@ def get_context_data(self, **kwargs):
         # TODO: This is inefficient because we're writing over the
         # already-generated choices. Ideally we should be passing these
         # into the formset constructor.
-        choices = projects_as_choices(reporting_period.get_projects())
+        projects = reporting_period.get_projects().filter(Q(organization=self.request.user.user_data.organization) | Q(organization=None))
```

Comment:
Without actually looking at get_projects, this makes me wonder if we should shift some of this filtering into it. It's already a manager, isn't it? Not saying we _should_ , since I haven't looked at it. But wondering.

Reply:
Yeah. I stopped short of pushing the filtering into `get_projects` because `ReportingPeriods` have no notion of users, so we'd have to pass either the user or the user's org into `get_projects`. I didn't love any of the options so I went with the least invasive one 😁